docs/joss/paper.md

```diff
@@ -0,0 +1,196 @@
+   index: 4
+ - name: Japan Meteorological Agency, Tsukuba, Japan
+   index: 5
+ - name: European Centre for Medium-Range Weather Forecasts, Reading, UK
+   index: 6
+ - name: University of Minnesota, Minneapolis, MN, USA
+   index: 7
+ - name: University of Melbourne, Australia
+   index: 8
+ - name: Australian National University, Canberra, Australia
+   index: 9
+   
+date: 14 September 2023
+bibliography: paper.bib
+
+---
+
+
+# Summary
+
+SpeedyWeather.jl is a library to simulate and analyse the global atmospheric
+circulation on the sphere. It implements several 2D and 3D
+models to solve the primitive equations with and without humidity (\autoref{fig:primitive}),
+the shallow water equations (\autoref{fig:swm}), or the barotropic vorticity equations
+with spherical harmonics. Several simple parameterizations for unresolved physical processes
+such as precipitation or the boundary layer are implemented, and new ones can
+be externally defined and passed as an argument to the model constructor.
+SpeedyWeather.jl is an intermediate-complexity general circulation model [@Kucharski2013]
+and research playground with an (almost) everything-flexible attitude.
+It can be thought of as a conceptual reinvention of the Fortran SPEEDY model [@Molteni2003]
+in the Julia programming language [@Bezanson2017].
+
+SpeedyWeather.jl internally uses three sub-modules `SpeedyTransforms`, `RingGrids`, and
+`LowerTriangularMatrices` to perform spherical harmonic transforms and interpolations
+between various grids and the spectral space. `RingGrids` discretize the sphere
+on iso-latitude rings and the spectral space is defined by the `LowerTriangularMatrices`
+of the spherical harmonic coefficients. These three modules are independently usable
+and therefore make SpeedyWeather.jl, beyond its main purpose of simulating the weather,
+also a library for the analysis of gridded data on the sphere.
+Running and analysing simulations can be interactively combined, enhancing user
+experience and productivity.
+
+The user interface of SpeedyWeather.jl is heavily influenced by
+the Julia ocean model Oceananigans.jl [@Ramadhan2020].
+A monolithic interface based on parameter files is avoided in favor of a
+library-style interface in which users write code to run models rather than
+merely supplying parameters and input arrays.
+A model is created bottom-up by first defining the discretization
+and any non-default model components with its respective parameters.
+All components are then collected into a single model object, which, once
+initialized, returns a simulation object that contains the entire model state,
+work arrays and parameters, that can be run, analysed or changed.
+While these steps can be written into a script for reproducibility,
+the same steps can be executed and interacted with one-by-one in
+Julia's read-evaluate-print loop (REPL). We thereby reach an interactivity
+far beyond a monolithic interface that is limited to the options provided.
```

Comment:
```suggestion
far beyond a monolithic interface that is limited to the options provided. At the same time, sensible default arguments enable even inexperienced users, to run simulations in just a few lines of code. 
```

Besides the modularity / bottom-up approach, I'd very briefly also mention how easy it is to actually get the model running. I think this is worth mentioning because starting a run - even with default arguments - of some old Fortran models can be quite cumbersome.

Reply:
Agree!